src/errorReporting.js

```diff
@@ -96,10 +96,19 @@ function getPendingPromise(state) {
   return state.getIn(['errorReporting', 'pendingPromise']);
 }
 
-export function serverSideError(error) {
-  // nodejs console truncates output by default
-  // https://nodejs.org/api/util.html#util_util_inspect_object_options
-  console.error(util.inspect(error, false, 10, true));
+export function serverSideError(queue) {
+  [].concat(queue).forEach((raw) => {
```

Comment:
what was the original function? this might be too much manipulation that isn't required? 🤔

Reply:
Original function was just logging the array as it is

`console.log(queue)`

which will output all errors in the queue in one line and omit the deep nested objects. This function will ensure that both the client and server errors print the same thing, one console line per error and the `metaData` instead of `otherData` which is not part of the schema

Reply:
ah, invoked with the queue of errors, not the error directly: good argument name change

I just remembered/re-read [`formatErrorReport`](https://github.com/americanexpress/one-app-ducks/blob/83c399693b826abb0c841443a0c0397c55346526/src/errorReporting.js#L39) and that the properties are grabbed off the Error instance, rather than preserving the error. This is needed in the client as Error doesn't serialize in JSON by default. On the server, IMO it'd be ideal to preserve the original Error instance (ex: retaining the name).

That might be a subsequent PR, in the meantime this looks good for now 👍

Reply:
sorry for the confusion & churn from my misunderstanding